templates/compose/home-assistant.yaml

```
@@ -0,0 +1,23 @@
+# documentation: https://www.home-assistant.io/
+# slogan: Open source home automation that puts local control and privacy first.
+# category: home-automation
+# tags: home-assistant, home-automation, smart-home, mqtt, zigbee, zwave
+# logo: svgs/home-assistant.svg
+# port: 8123
+
+services:
+  homeassistant:
+    image: homeassistant/home-assistant:${IMAGE_TAG:-stable}
+    environment:
+      - SERVICE_URL_HOMEASSISTANT_8123
+      - TZ=Europe/Berlin
```

Hardcoding TimeZone like this is not a good idea because people from other timezone will have to manually edit this hardcoded timezone.

```suggestion
      - TZ=${TZ:-Europe/Berlin}
```